### ISSUE TITLE
Sweden: Includes everything with P1435 

### ISSUE THREAD

**Issue by Abbe98** [hatnote/monumental#19]
*For now an data issue*

In Sweden only a limited subset of items that can have P1435 heritage designation stated is used in WLM, Monumental currently queries all of them.

Example:
The church "Floda Kyrka" has in Wikidata two items with P1435 one is the facility and one is the actual building, in Sweden only facilities has previously been used in WLM. Both items have heritage designation specified.

Other types of heritage sites are limited to other types of subset filters.

@lokal-profil Wikidata items needs some type of additional statement so we can distinguish items that should be a part of WLM and those which shouldn't.

**Comment by lokal-profil** [comment]
Actually any protected building (and archaeological site) qualifies for WLM in Sweden even though only some of them are listed in the lists on Wikipedia.

The main problem with e.g. a building being used instead of the complex is that it becomes harder to verify that the image does indeed qualify. at the same time the building id is even more valuable than the complex id (if used right).

I think one solution would be to have a bot patrol the images with that template and add the complex id if only the building id is used. That could be extended to patrol for building ids used as complex ids or invalid id's in general.

**Comment by Abbe98** [comment]
@lokal-profil but currently a image uploaded through a building would get a template like `{{BBR|a|<building-id>}}` which would end up being invalid. So a patrolling bot would need to validate and fix all those too in addition to adding the complex id.

**Comment by lokal-profil** [comment]
@Abbe98 True. And with {{BBR|a|...}} being used there is no easy trivial way of patrolling the images either.
